docs/creators/creators.yml

```diff
@@ -478,3 +478,14 @@
   for_hire: true
   hiring: false
   portfolio: true
+- name: App Design
+  type: agency
+  description: >-
+    My name is Oliver. I am a web developer and app designer.
+  location: Canada
+  website: "https://appdesign.dev/"
+  github: "https://github.com/appdesigndev"
+  image: images/app-design-dev.jpeg
```

Comment:
This image wasn't added, so I do need to revert this PR for now ( https://github.com/gatsbyjs/gatsby/pull/21223 ). Please re-open and the image ;)

Reply:
Oops

Reply:
it say: "You’re not authorized to merge this pull request." I need Code owner review required, thanks :)